airqo-platform/AirQo-frontend
# [Collocation] Ft monitor report

#### Summary of Changes (What does this PR do?)

- Added Reports page
- Device report page with data completeness table, intersensor & intrasensor correlation charts
- Created reusable chart component called correlation chart
- Created reusable chart box skeleton
- Created reusable pollutant component
- Added e2e tests for collocation feature

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Changes can be found in deploy preview. To see reports page, go to the collocate page and on the device status table, click on the horizontal dots of a device that has data(typically devices that have passed are the ones with data). On the reports, you can click read more to see the monitor reports page which has the data completeness results etc

#### What are the relevant tickets?
- [AN-320](https://airqoteam.atlassian.net/browse/AN-320)

#### Screenshots (optional)
![image](https://user-images.githubusercontent.com/46527380/230343836-2df35f9e-1e83-4b8e-beaa-01e01cadb787.png)
![Screenshot from 2023-03-26 10-32-28](https://user-images.githubusercontent.com/46527380/227761817-94919be4-9ccf-43a7-a481-c47659aa64fa.png)
![Screenshot from 2023-03-26 10-32-12](https://user-images.githubusercontent.com/46527380/227761819-68c4959e-0495-4e70-b947-b2405a6389c7.png)
![image](https://user-images.githubusercontent.com/46527380/227761835-9501103a-d39f-43fd-bb4d-9d8598c5ef26.png)


[AN-320]: https://airqoteam.atlassian.net/browse/AN-320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
